### PR TITLE
DDCE-4354 Remove use of deprecated Crypto stuff

### PR DESCRIPTION
**Pull request by gastonracui**
# DDCE-????

Please include a summary / description of the change and which issue it fixes.  Include any relevant user needs, context or links to other PRs related to this PR (eg. acceptance tests, environment config).

## Checklist PR Raiser
- [ ]  I've updated the README if there are functionality changes
- [ ]  I've executed the project script to check Dependencies and update them if they're non-breaking changes
- [ ]  I've executed Scalastyle and fixed any warnings that can be resolved quickly
- [ ]  I've executed the acceptance test pack locally to ensure there are no functional regressions (Performance tests where relevant)
- [ ]  I've included appropriate tests with any code I've added (Unit, Integration), and it meets Coverage
- [ ]  I've squashed my commits - including the JIRA issue number in the commit message
- [ ]  I've contacted a BA to create a task if any of these actions would result in a larger piece of work

### CODE REVIEW

**Review by frikit — CHANGES_REQUESTED** [review]
- Please update deps (except mongo one)

File: project/AppDependencies.scala [hunk 1]
```diff
@@ -3,16 +3,16 @@ import play.core.PlayVersion.current
...
 
   private lazy val compile: Seq[ModuleID] = Seq(
     "commons-validator"            % "commons-validator"           % "1.7",
-    "uk.gov.hmrc"                  %% "play-frontend-hmrc"         % "7.13.0-play-28",
+    "uk.gov.hmrc"                  %% "play-frontend-hmrc"         % "7.14.0-play-28",
     "uk.gov.hmrc"                  %% "bootstrap-frontend-play-28" % bootstrapPlayVersion,
     "uk.gov.hmrc"                  %% "http-caching-client"        % "10.0.0-play-28",
     "uk.gov.hmrc.mongo"            %% "hmrc-mongo-play-28"         % hmrcMongoPlayVersion,
-    "commons-codec"                % "commons-codec"               % "1.15",
+    "commons-codec"                % "commons-codec"               % "20041127.091804",
```
Comment (frikit) [inline project/AppDependencies.scala:15]: latest version is 1.16 please check maven repository https://mvnrepository.com/artifact/commons-codec/commons-codec

**Review by frikit — CHANGES_REQUESTED** [review]

File: project/AppDependencies.scala (hunk 1 above)
Comment (gastonracui) [reply project/AppDependencies.scala:15]: I tried with that one but failed to update the sbt

**Review by frikit — CHANGES_REQUESTED** [review]
I like the allowfilter change seems to be safe but you forgot to delete this:
```
filters {
  allowlist {
    ips = ""
    destination = "https://www.gov.uk"
    excluded = ""
  }
}
```

**Pull request merged**